src/test/java/teammates/test/cases/browsertests/FeedbackContributionQuestionUiTest.java

```diff
@@ -71,12 +71,31 @@ public void testCustomizeOptions() {
 
         //no question specific options to test
 
-        ______TS("CONTRIB: set visibility options");
+        ______TS("CONTRIB: verify only valid visibility options are visible");
 
-        feedbackEditPage.enableOtherVisibilityOptionsForNewQuestion();
-        //TODO: click and ensure can see answer for recipients,
-        //giver team members, recipient team members
-        //are always the same. (under visibility options)
+        assertTrue(feedbackEditPage
+                .isVisibilityDropdownOptionHiddenForNewQuestion("ANONYMOUS_TO_RECIPIENT_AND_INSTRUCTORS"));
+        assertTrue(feedbackEditPage
+                .isVisibilityDropdownOptionHiddenForNewQuestion("ANONYMOUS_TO_RECIPIENT_VISIBLE_TO_INSTRUCTORS"));
+        assertFalse(feedbackEditPage
+                .isVisibilityDropdownOptionHiddenForNewQuestion("ANONYMOUS_TO_RECIPIENT_AND_TEAM_VISIBLE_TO_INSTRUCTORS"));
+        assertFalse(feedbackEditPage.isVisibilityDropdownOptionHiddenForNewQuestion("VISIBLE_TO_INSTRUCTORS_ONLY"));
+        assertTrue(feedbackEditPage.isVisibilityDropdownOptionHiddenForNewQuestion("VISIBLE_TO_RECIPIENT_AND_INSTRUCTORS"));
+        assertTrue(feedbackEditPage.isVisibilityDropdownSeparatorHiddenForNewQuestion());
+        assertTrue(feedbackEditPage.isVisibilityDropdownOptionHiddenForNewQuestion("OTHER"));
+
+        ______TS("CONTRIB: verify correct output params for selected visibility option");
+
+        // default (first option)
```

Comment:
For better user experience, can we just select a option by default? (and I see here you say even the option is `Please select ....`, the option is still the first one)